docs/CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ The following tip changes can be tested by building VictoriaMetrics components f
   - `vm_vmselect_concurrent_requests_current` - the current number of concurrently executed requests
   - `vm_vmselect_concurrent_requests_limit_reached_total` - the total number of requests, which were put in the wait queue when `-search.maxConcurrentRequests` concurrent requests are being executed
   - `vm_vmselect_concurrent_requests_limit_timeout_total` - the total number of canceled requests because they were sitting in the wait queue for more than `-search.maxQueueDuration`
+* FEATURE: use `-httpAuth.*` flags as a rollback for `-*AuthKey` flags when `-httpAuth.*` flags are set. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3060).
```

Comment:
```suggestion
* FEATURE: protect all the http endpoints with `-httpAuth.*` command-line flag. Previously endpoints protected by `-*AuthKey` command-line flags weren't protected by `-httpAuth.*`. This could complicate the proper security setup. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3060).
```